vital/types/bounding_box.h

```diff
@@ -237,22 +237,20 @@ bool operator== ( bounding_box<T> const& lhs, bounding_box<T> const& rhs )
   return false;
 }
 
-
 /**
- * @brief Inequality operator for bounding box
+ * @brief Inquality operator for bounding box
```

Comment:
Looks like you changed from proper spelling to error ... assuming you are speaking about inequality.